docs/agda-spec/src/Ledger/PParams.lagda

```diff
@@ -86,53 +96,74 @@ record PParams : Type where
 \end{code}
 \emph{Network group}
 \begin{code}
```

Comment:
Could you please indicate the types of these two new protocol parameters?

Reply:
They are both byte counts

Reply:
Fixed in 7be352fcf49b9491f6a33fb457136118c69fdc47:

https://github.com/IntersectMBO/ouroboros-consensus/blob/7be352fcf49b9491f6a33fb457136118c69fdc47/docs/agda-spec/src/Ledger/PParams.lagda#L120-L121

Please note that other changes have been introduced, since I have actually synchronized this file with its corresponding source in the Agda ledger spec.